src/redux/reducers/currentProjectReducer.ts

```diff
@@ -21,34 +21,32 @@ export const reducer = (state: IProject = null, action: AnyAction): IProject =>
             return null;
         case ActionTypes.LOAD_PROJECT_SUCCESS:
             return { ...action.payload };
-        case ActionTypes.LOAD_PROJECT_ASSETS_SUCCESS:
```

Comment:
When loading assets from the asset provider it no longer saves them in the project assets by default.  This is a perf optimization in the event of a large number of assets to help reduce project load time.